### PR TITLE
Add separate layout for teacher

### PR DESCRIPTION
**Pull request by erinnachen**
- Added navigation bar with styling from existing Roots application
- Students do not have a navigation bar
- Links do not yet go anywhere. 
- Added a route '/logout' to enable students to logout from navigating to page

@jedschneider We had talked about the hidden logout path, I placed it in the sessions controller, since it felt like authentication was all grouped in there. Any thoughts?

### CODE REVIEW

File: app/controllers/users/sessions_controller.rb [hunk 1]
```diff
@@ -1,7 +1,10 @@
 class Users::SessionsController < Devise::SessionsController
-  after_action :verify_authorized, except: [:new, :create, :destroy]
+  after_action :verify_authorized, except: [:new, :create, :destroy, :logout]
 
   def after_sign_in_path_for(resource)
     resource.after_sign_in_path
   end
+
+  def logout
```
Comment (jedschneider) [inline app/controllers/users/sessions_controller.rb]: devise implements its own logout action and view. you might look into that first.

**Comment by jedschneider** [comment]
we had talked about using the application layout for the teacher view, since the compass was the only view on the student side, and so, generally is the exception to the rule. I guess maybe you had second thoughts on that?

File: app/controllers/users/sessions_controller.rb (hunk 1 above)
Comment (erinnachen) [inline app/controllers/users/sessions_controller.rb]: @jedschneider devise doesn't look like it implements a view for logout... I'm going to leave this for now unless you have a resource to point me to.

**Comment by erinnachen** [comment]
We did have this discussion and for some reason decided that because we only have one teacher page versus a root page and a compass page that I should make the layout in the dashboard controller. Will change that now, just a brain fart on my part.

**Comment by damwhit** [comment]
Looks good to me.  I'm merging so that we can throw some styles on it.

**Pull request merged**